indico/modules/events/layout/util.py

```diff
@@ -198,7 +201,7 @@ def _check_menu(event):
                       joinedload('children').load_only('id', 'parent_id', 'name', 'position')))
 
     existing = {entry.name: entry for entry in query}
-    pos_gen = count(start=(max(x.position for x in existing.itervalues() if not x.parent)))
+    pos_gen = count(start=(max(x.position for x in existing.itervalues() if not x.parent) + 1))
```

Comment:
Nice catch.

Reply:
didn't really cause problems, just looked a bit ugly when having the same position displayed twice in the menu editing view